app/controllers/application_controller/miq_request_methods.rb

```diff
@@ -158,10 +163,19 @@ def set_pre_prov_vars
     @edit[:vm_sortcol] ||= "name"
     @edit[:prov_type] = "VM Provision"
     @edit[:hide_deprecated_templates] = true if request.parameters[:controller] == "vm_cloud"
+
     unless %w(image_miq_request_new miq_template_miq_request_new).include?(params[:pressed])
+      report_name = request.parameters[:controller] == "vm_cloud" ? "ProvisionCloudTemplates.yaml" : "ProvisionInfraTemplates.yaml"
+      path_to_report = ManageIQ::UI::Classic::Engine.root.join("product", "views", report_name).to_s
+      @view = MiqReport.new(YAML.safe_load(File.open(path_to_report), [Symbol]))
```

Comment:
Still trying to figure out why this is needed? Why not just use `get_view`, the same as everywhere else (even in provisioning when called the second time now)?

Reply:
I guess it can be used (didn't try yet), but I just need to load report and `get_view` does a lot of other things which are not necessary to do here.

Reply:
Makes sense :) Just something to watch for, as we have now 2 sources of report for the GTL.